src/app/view/components/_common/iframe/iframe-component/iframe.component.spec.ts

```diff
@@ -74,4 +72,49 @@ describe('IframeComponent', () => {
         expect(comp.iframeElement.nativeElement.contentWindow.location.reload).toHaveBeenCalledTimes(1);
     });
 
+    describe('bind iframe events', () => {
+        beforeEach(() => {
+            comp.iframeElement.nativeElement = {
+                contentWindow: {
+                    document: {
+                        body: {
+                            innerHTML: '<html></html>'
+                        },
+                        addEventListener: jasmine.createSpy('docAddEventListener'),
+                        removeEventListener: jasmine.createSpy('docRemoveEventListener')
+                    },
+                    addEventListener: jasmine.createSpy('addEventListener'),
+                    removeEventListener: jasmine.createSpy('removeEventListener')
+                }
+            };
+        });
+
+        it('should remove listener on load', () => {
+            iframeEl.triggerEventHandler('load', {});
+
+            expect(comp.iframeElement.nativeElement.contentWindow.removeEventListener).toHaveBeenCalledWith(
+                'keydown',
+                jasmine.any(Function)
+            );
+            expect(comp.iframeElement.nativeElement.contentWindow.document.removeEventListener).toHaveBeenCalledWith(
+                'ng-event',
+                jasmine.any(Function)
+            );
+        });
+
+        it('should add listeners load', () => {
+            iframeEl.triggerEventHandler('load', {});
+
+            expect(comp.iframeElement.nativeElement.contentWindow.addEventListener).toHaveBeenCalledWith(
+                'keydown',
+                jasmine.any(Function)
+            );
+            expect(comp.iframeElement.nativeElement.contentWindow.document.addEventListener).toHaveBeenCalledWith(
+                'ng-event',
+                jasmine.any(Function)
+            );
+        });
+    });
+
+    xit('should trigger keydown', () => {});
```

Comment:
what happend with this?

Reply:
Don't know how to test it at this point.